src/pages/App.svelte

```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  // @ts-ignore no declarations
```

Comment:
There's some way to define a .d.ts file to get TS declarations for external libraries, but haven't figured out exactly how, and it's not too important -- we're just calling this one init function